src/visualization/plot-recipes.jl

```diff
@@ -115,3 +115,20 @@ end
     markerstrokewidth --> 1.5
     model.space.graph
 end
+
+"""
+    abm_plot_on_graph(model::ABM{<: GraphSpace}; ac, as, am, kwargs...)
```

Comment:
What does this do? plotabm already handles graphs?

Reply:
`plotabm` has no reason to exist anymore for gridspace and continuousspace, and the name was too close to `abm_plot`. I thought having an explicit name for graph plots is better for the user.

Reply:
If you see the new docs on visualization, `plotabm` is not expanded anywhere

Reply:
I added it in to the plot discussions (the bottom of the docs were not finished). Change the recipe names, sure, but just because we don't want to use Plots.jl anymore doesn't mean everyone wants to.